lib/Model.ts

```diff
@@ -692,6 +693,52 @@ export class Model<TDocument extends { _id?: any }, TInstance> {
         });
     }
 
+    private _mapReduce<Key, Value>(instanceType: InstanceImplementation<MapReducedDocument<Key, Value>, any>, options: MongoDB.MapReduceOptions): Bluebird<void> {
```

Comment:
Yes, I got the point. Moved the private function into the public functions branch.